.styleci.yml

```diff
@@ -0,0 +1,8 @@
+preset: psr2
+
+enabled:
+  - long_array_syntax
+  - duplicate_semicolon
+
+disabled:
+  - short_array_syntax
```

Comment:
You don't need to disable this when using the psr2 present. It's already disabled.